lms/templates/logout.html

```diff
@@ -0,0 +1,23 @@
+{% extends "main_django.html" %}
```

Comment:
I feel like someone is going to yell about safe templates and how we should be using Mako for all new templates, but I'm not totally certain about that.